spec/factories/fee_schemes.rb

```diff
@@ -29,6 +29,12 @@
       version { 12 }
     end
 
+    trait :agfs_thirteen do
```

Comment:
As we now have a new agfs scheme, do we need to apply an end date to the previous agfs scheme (`trait :agfs_twelve`)?